src/main/java/org/truffleruby/options/CommandLineParser.java

```diff
@@ -557,8 +545,12 @@ private String grabOptionalValue() {
         return null;
     }
 
+    private CommandLineException notImplemented(String option) {
+        return new CommandLineException(String.format("the %s option is not implemented", option));
+    }
+
     public static void printHelp(PrintStream out) {
-        out.println("Usage: truffleruby [switches] [--] [programfile] [arguments]");
+        out.println("Usage: ruby [switches] [--] [programfile] [arguments]");
```

Comment:
And that's not the executable name:

```
$ rbenv shell jruby-9.1.12.0
$ ruby -Z -e 3
jruby: unknown option -Z
```

Ok I'll switch to `truffleruby` for both.